MapboxNavigation/FreeDriveDebugger.swift

```diff
@@ -0,0 +1,132 @@
+import UIKit
+import Mapbox
+import MapboxCoreNavigation
+
+class CLLToMGLConverterLocationManager: NSObject, MGLLocationManager, CLLocationManagerDelegate {
```

Comment:
Can you describe the purpose of this class? It’s a private class, though some of its members are marked public. It’s also in a file called FreeDriveDebugger.swift, which seems temporary.

Reply:
Initially I used it from the Example app, that's why it needed to be public, for now there is no reason for the methods to be public.

The class by itself is just a converter from one location manager protocol to another. The problem is that there are `NavigationLocationManager` and `MGLLocationManager` protocols which can't be implemented by one class at the same time. `FreeDriveLocationManager` conforms to `NavigationLocationManager` (and therefore to `CLLocationManager`). To show mapped locations on the map we need to provide MGLLocationManager, the `CLLToMGLConverterLocationManager` converts `CLLocationManager` to `MGLLocationManager`

Reply:
I’ve made the class public, since the whole point of this feature is that applications can integrate a replacement for CLLocationManager into a map view at the application level. The file and class still need to be renamed.